sprite.go

```diff
@@ -204,10 +204,6 @@ func applySpriteProps(dest *Sprite, v specsp) {
 	if style, ok := v["rotationStyle"]; ok {
 		dest.rotationStyle = toRotationStyle(style.(string))
 	}
-	if _, ok := v["currentCostumeIndex"]; ok {
```

Comment:
This code is removed, but member `currentCostumeIndex` is not removed.
And you need check all games based on spx are not broken: https://github.com/goplus/spx?tab=readme-ov-file#games-powered-by-spx